web/src/styles.css

```diff
@@ -111,15 +111,15 @@ line, path {
 }
 
 .country-panel{
-    overflow-y: auto;
+    overflow-y: scroll;
     padding-right: calc(2rem - 6px);
     padding-left: 2rem;
     margin: 0;
     -webkit-overflow-scrolling: touch;
 }
 .country-panel-wrap {
     position: relative;
-    padding-top: 160px;
+    padding-top: 200px;
```

Comment:
now that the header is included, more padding must be applied to left panel content